src/test/regress/upgrade/README.md

```diff
@@ -42,7 +42,10 @@ How the postgres upgrade test works:
 
 - Temporary folder `tmp_upgrade` is created in `src/test/regress/`, if one exists it is removed first.
 - Database is initialized and citus cluster is created(1 coordinator + 2 workers) with old postgres.
-- `before_pg_upgrade_schedule` is run with `pg_regress`. This schedule does not drop any tables or data so that we can verify upgrade.
+- `before_pg_upgrade_schedule` is run with `pg_regress`. This schedule sets up any
+  objects and data that will be tested for preservation after the upgrade. It
+  also runs all the tests that will be run after the upgrade, to make sure their
```

Comment:
this line is not up to date with our changes that the after schedule is now executed twice.